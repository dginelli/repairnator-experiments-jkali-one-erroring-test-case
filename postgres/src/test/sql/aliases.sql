CREATE TABLE aliases (id serial8 NOT NULL PRIMARY KEY, title text);
CREATE INDEX idxaliases ON aliases USING zombodb (zdb(tableoid, ctid), zdb(aliases)) WITH (url='http://localhost:9200/', alias='the_alias');
INSERT INTO aliases VALUES (DEFAULT, 'one');
INSERT INTO aliases VALUES (DEFAULT, 'two');
INSERT INTO aliases VALUES (DEFAULT, 'three');
SELECT * FROM aliases WHERE zdb(tableoid, ctid) ==> '' ORDER BY id;
SELECT zdb_estimate_count('aliases', '');
DROP TABLE aliases;