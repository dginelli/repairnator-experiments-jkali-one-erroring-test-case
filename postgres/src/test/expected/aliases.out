CREATE TABLE aliases (id serial8 NOT NULL PRIMARY KEY, title text);
CREATE INDEX idxaliases ON aliases USING zombodb (zdb(tableoid, ctid), zdb(aliases)) WITH (url='http://localhost:9200/', alias='the_alias');
INSERT INTO aliases VALUES (DEFAULT, 'one');
INSERT INTO aliases VALUES (DEFAULT, 'two');
INSERT INTO aliases VALUES (DEFAULT, 'three');
SELECT * FROM aliases WHERE zdb(tableoid, ctid) ==> '' ORDER BY id;
 id | title 
----+-------
  1 | one
  2 | two
  3 | three
(3 rows)

SELECT zdb_estimate_count('aliases', '');
 zdb_estimate_count 
--------------------
                  3
(1 row)

DROP TABLE aliases;
